Cargo.toml

```diff
@@ -53,6 +54,7 @@ cockroach_db = []
 kwok = []
 
 [dependencies]
+parse-display = { version = "0.10.0", optional = true, default-features = false, features = [] }
```

Comment:
`0.10.0` has changed MSRV
Let's use `0.9.1` for now, because `testcontainers` uses this too.

I'll update it once MSRV of `testcontainers` is bumped

Reply:
have added a commit (includes a `TODO` comment why this is not up to date and when this can be updated ^^)